packages/twenty-docker/prod/twenty/inject-runtime-env.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+echo "Generating env-config.js file from runtime environment variables..."
```

Comment:
copy/pasting this code from somewhere else no? can we share code instead?

Reply:
Yeah it comes from here https://github.com/twentyhq/twenty/blob/main/packages/twenty-front/scripts/inject-runtime-env.sh
I first had planned to deprecate it when we would clean the front container, but I just noticed that it's also used in the package.yml of the front, so it makes sense to keep it there, I'll adapt the script to handle custom path